### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by imhamad**
Added some more clarity to 2nd line for better flow and readability.

To be more specific, irrespective is replaced by regardless for better readibility.

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: "If you want to make your software applications more sustainable an
 
 Do you want to be a part of the green software movement?
 
-If you want to make your software applications more sustainable and build a career in green software, then this training is for you. It will teach you how to build, maintain and run greener applications irrespective of the application domain, industry, organization size or type, programming language, or framework.
+If you want to make your software applications more sustainable and build a career in green software, then this training is for you. It will teach you how to build, maintain, and run greener applications, regardless of the application domain, industry, organization size or type, programming language, or framework.
```
Comment (franziska-warncke) [inline docs/index.mdx:10]: somehow, I would not add extra comma here after maintain and applications, but I like regardless more than irrespective.  .

Comment (imhamad) [reply docs/index.mdx:10]: Sweet! Let's add regardless and remove the comma @franziska-warncke

**Review by imhamad — COMMENTED** [review]
As requested by @franziska-warncke the comma "," has been eradicated and the word regardless stayed as is,

File: docs/index.mdx (hunk 1 above)
Comment (imhamad) [reply docs/index.mdx]: Done @franziska-warncke , happy for you to check and merge accordingly

**Review by franziska-warncke — APPROVED** [review]
Looks good 🌻

**Comment by franziska-warncke** [comment]
@russelltrow Please have a look and merge it! Thanks :)

**Comment by imhamad** [comment]
Super sweet! Thank you

From: Franziska Warncke ***@***.***>
Date: Tuesday, 23 July 2024 at 1:05 PM
To: Green-Software-Foundation/learn ***@***.***>
Cc: Hamad ***@***.***>, Author ***@***.***>
Subject: Re: [Green-Software-Foundation/learn] Update index.mdx (PR #232)

@franziska-warncke approved this pull request.

Looks good 🌻

**Pull request merged**